examples/basics/plotting/grid_uneven_col.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+   c0    c1    c2
+r0 +-----+-----+-----+
+   |  1  |  2  |  2  |
+r1 +-----+-----+-----+
+   |  3  |  3  |  4  |
+   +-----+-----+-----+
```

Comment:
In other words, a `col_span=2` should set the horizontal stretch = 2. Does that not happen automatically right now?